app/main/controllers/template/rtmedia-filters.php

```diff
@@ -282,6 +282,9 @@ function replace_src_with_transcoded_file_url( $html, $rtmedia_media ) {
 		$final_file_url = wp_get_attachment_url( $attachment_id );
 	}
 
+    //Add timestamp to resolve conflict with cache media.
+    $final_file_url = $final_file_url . '?' . time();
+    
 	return preg_replace( '/src=["]([^"]+)["]/', "src=\"$final_file_url\"", $html );
```

Comment:
Since `'?' . time()` has to be added to the src anyway, can we add it directly to `src` attribute?

Reply:
yes, added.